webapp/views.py

```diff
@@ -307,22 +307,28 @@ def api_v1_players_team(teamcode):
     with open(PLYR_MSTR_I, 'r') as playerMaster:
         plyr_mstr = json.load(playerMaster)
 
```

Comment:
These changes are to always display team name on player stats page and stop using vs and at to determine home and away.